cabal.project

```diff
@@ -0,0 +1,2 @@
+packages:
+  .
```

Comment:
Why do we need cabal.project if there is only one package?

Reply:
Good point -- At the moment, we don't need it, so I've removed it